scripts/ai/controllers/CombineController.lua

```diff
@@ -124,59 +123,4 @@ end
 function CombineController:isDroppingStrawSwath()
     return self.combineSpec.strawPSenabled
 end
--------------------------------------------------------------
```

Comment:
See the comment above about the chopper controller.
This logic applies mostly indirect to the combine Spec and should be placed here.